AutomationHelpers/src/Modules/ReportToPDFModule.cs

```diff
@@ -282,6 +282,7 @@ internal void Cleanup()
 			}
 		}
 
+#pragma warning disable CS0618

```

Comment:
Why not put the pragma directly around the UseScreenshotFolder?

Reply:
Isn't that ugly to put the Pragma to the method directly?

Reply:
I would limit it to the minimum context so we see other warnings